.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+cache: pip
+python:
+    - "2.7"
+    - "3.4"
+    - "3.5"
+    - "3.5-dev"
+    - "3.6"
+    - "3.6-dev"
+    - "3.7-dev"
+matrix:
+    allow_failures:
+        - python: "2.7"
+        - python: "3.4"
+        - python: "3.7-dev"
+addons:
+    apt:
+        packages:
+            - dssp
+env:
+    - VERMOUTH_TEST_DSSP=mkdssp
+install:
+    - pip install pylint
+    - pip install --upgrade .
+script:
+    - pytest -vv vermouth
+after_script:
+    - pylint vermouth
+    - pylint bin/martinize2
```

Comment:
You could argue that this should be part of the test, and not just a nice output to see. Either way, we'll probably need a custom .pylintrc. 
I guess pytest has a pylint plugin, to wrap it all in one.
Also, do we want to look at test coverage?

Reply:
There are way too many messages at the moment. We need a .pylintrc AND a lot of fixing before we can use it as a test.

Reply:
Let's keep it in mind for the future. 
And yes, if I recall correctly, PyLint is rather... How do I phrase this... Strict.